src/components/MainHeading/index.jsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import './main-heading.css';
 
-const MainHeading = ({ text }) => <h3 className="main-head">{text}</h3>;
+const MainHeading = ({ text, className }) => (
+  <h3 className={className}>{text}</h3>
```

Comment:
retrieve the MainHeading style file and write // className={`main-head ${className}`}
instead of // className={className}